### ISSUE TITLE
Shell commands allow for invalid Property types to be set

### ISSUE THREAD

**Issue by ddanielr** [apache/accumulo#3145]
**Describe the bug**

When using the accumulo shell to set properties, a user is able to set an invalid property value for value types such as Boolean and Port. 

Example: 
This property is a boolean type.
https://github.com/apache/accumulo/blob/fbe1ee8cfc98f788329208419d682429c90784c2/core/src/main/java/org/apache/accumulo/core/conf/Property.java#L438-L439

However, it can be set to an integer value.
```        
root@uno> config -s tserver.monitor.fs=345                                      
root@uno> config -f tserver.monitor.fs                                          
-----------+--------------------------------------------------------------------+-----------------------------------------------------------------------------------                       
SCOPE      | NAME                                                               | VALUE                            
-----------+--------------------------------------------------------------------+-----------------------------------------------------------------------------------                       
default    | tserver.monitor.fs ............................................... | false
system     |    @override ..................................................... | 345                                                                                                      
-----------+--------------------------------------------------------------------+-----------------------------------------------------------------------------------
```

**Versions (OS, Maven, Java, and others, as appropriate):**
 - Affected version(s) of this project: 2.1.x, 1.10.x
 - OS: Centos7
 - Others:

**To Reproduce**
Steps to reproduce the behavior (or a link to an example repository that reproduces the problem):
1. Start accumulo Shell
2. Use the Config Command to find a Boolean or Port Property Type `config -f <property>`
3. Use the `config -s <property>=<invalid value>` command to set that type to an invalid value
4. Use `config -f <property>` to show the invalid value is set

**Expected behavior**
The Shell tool should check the property value and error on invalid values instead of allowing them to be set. 

**Additional context**
I assume that all versions of accumulo have this bug, but I have only checked against 1.10, 2.1, and main.

**Comment by ctubbsii** [comment]
Since the shell should be calling public API for this, it should be possible for us to validate updated configuration on the server-side, before we set it, and then either store or reject the updated configuration.

**Comment by ddanielr** [comment]
Changed my proposed solution to validate properties on the server-side and solve this issue for all clients.

**Comment by dlmarion** [comment]
Closed by #3177
